tox.ini

```diff
@@ -197,7 +203,7 @@ deps =
 commands_pre =
   python -m pip install -e {toxinidir}/opentelemetry-api[test]
   python -m pip install -e {toxinidir}/opentelemetry-semantic-conventions[test]
-  python -m pip install {toxinidir}/opentelemetry-python-contrib/opentelemetry-instrumentation
+  python -m pip install -e {toxinidir}/opentelemetry-instrumentation[test]
```

Comment:
It doesn't look like `opentelemetry-instrumentation` extra `test` has anything?

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/08daa9d194ee9484fcee506f145901e551aecd10/opentelemetry-instrumentation/setup.cfg#L55-L56